note applicability of future scheduling algorithms (#538)
quicwg/multipath: draft-ietf-quic-multipath.md

to be handled by the application. But this document does not prevent future extensions from
defining mechanisms to cope with the remaining scenarios.

> Further, this document does not specify detailed scheduling algorithms that define

Author: Proposed by Gorry:
```suggestion
Further, this document does not specify scheduling algorithms that define
```

how multiple, simultaneously open paths are used to send packets.
As these differ depending on application requirements,
only some basic implementation guidance is discussed in {{impl-consideration}}.

> However, scheduling algorithms, that may be specified by the IETF in future,

Author: ```suggestion
However, scheduling algorithms that may be specified by the IETF in future
```

can be used with the multipath extension specified in this document.

> 

Reviewer: Too many commas? Copy editors insist on no comm before "that", and the one after future is not useful.

Author: I thought they help but removed them now.

Specifically, while failover between Wi-Fi
and mobile networks is a well-known multipath use case,

> it only uses two paths at the same time temporarily

Author: ```suggestion
it only temporarily uses two paths at the same time
```

to be handled by the application. But this document does not prevent future extensions from
defining mechanisms to cope with the remaining scenarios.

Further, this document does not specify scheduling algorithms that define
how multiple, simultaneously open paths are used to send packets.
As these differ depending on application requirements,
only some basic implementation guidance is discussed in {{impl-consideration}}.
This extension can be used with different scheduling algorithms that,
e.g., can range from support for failover to simulatenous

> use of the aggreate capacity across all active paths.

Author: This edit fixes a nit and aligns terminology (as active paths include paths with issued CIDs that are not open yet):
```suggestion
use of the aggregated capacity across all open paths.
```

> There are IETF specifications for path fail-over, but

Author: Do we really need to mention that there is a failover spec? As I said earlier I don’t think RFC7829 is necessarily relevant here as it doesn’t really assume that both paths are used simultaneously. If we want to keep this however I think we need to give a reference…?

```suggestion
```

Reviewer: I agree that we should strike out that sentence. I was almost ready to commit the suggestion, but will let @mirjak do that once consensus is assessed.

I did read RFC7829, and I don't think citing it would be useful. The multipath extension does not create a new path fail-over capability, because that capability is already defined in RFC 9000. For example, the concern about simultaneous failover creating a thundering herd issue applies equally with and without the multipath extensions to RFC 9000.

Reviewer 2: I hoped mentioning path-failover strengthened the position for making the QUIC MP Extension a PS. It did at least figure when MP items were chartered in the past. It is OK to remove this.

Reviewer 2: I agree RFC7829 is not about concurrent usage of paths for data - I see this as opening two paths and can keep them alive until a path change is needed.

> there are currently no IETF standards-track specifications for simultaneously

Author: Can we maybe just remove „standards-track“? Because this is also true for none standards-track specs.

```suggestion
there are currently no IETF specifications for simultaneously
```

Reviewer 2: RFC 6356, RFC8382 and RFC 8699 are all experimental specifications for scheduling, albeit not targeting QUIC, but they are all EXP.

Author: These RFCs are all about congestion control but do not specify scheduling.

> (concurrently) using the aggregated capacity of a set of active paths.

Author: Do we need to say „aggregated capacity“ here again?

This would be less wordy:
```suggestion
(concurrently) using multiple paths.
```

Reviewer: +1

Reviewer 2: Sure, I could live without "aggregated capacity" here if the word concurrently is understood to say that.
